prime-router/src/main/kotlin/cli/SenderUtilsCommand.kt

```diff
@@ -67,27 +72,39 @@ class AddPublicKey : SettingCommand(
             return
         }
         val jwk = SenderUtils.readPublicKeyPemFile(publicKeyFile)
-        jwk.kid = if (kid.isNullOrEmpty()) settingName else kid
+        jwk.kid = if (kid.isNullOrEmpty()) orgName else kid
 
-        val origSenderJson = get(environment, oktaAccessToken, SettingType.SENDER, settingName)
-        val origSender = jsonMapper.readValue(origSenderJson, Sender::class.java)
+        val origOrganizationJson =
+            get(environment, oktaAccessToken, SettingType.ORGANIZATION, orgName)
+        val origOrganization = jsonMapper.readValue(origOrganizationJson, Organization::class.java)
 
-        if (!Scope.isValidScope(scope, origSender)) {
-            echo("Sender full name in scope must match $settingName.  Instead got: $scope")
+        if (!Scope.isValidScope(scope, origOrganization)) {
```

Comment:
I am a bit confused, if you are ripping out all of the sender to replace it with organization, how are you keeping backwards compatibility?

Reply:
So the backwards compatibility is maintained for where the keys are read from, i.e. a key on a sender can still be used for authenticating. 

The ability to add keys to a sender though is not getting maintained.

Reply:
@JessicaWNava Small bump if there was anything else I could clarify/update here

Reply:
Nope, sorry, just got sucked into other things. Thank you for the clarification!